crates/biome_js_formatter/tests/specs/js/module/decorators/class_members_call_decorator.js.snap

```diff
@@ -211,7 +211,8 @@ class Foo {
 	@decorator.method(value) /*before*/
 	/*after*/ method() {}
 	@decorator.method(value) /*before*/
-	async /*after*/ method() {}
+	/*after*/
+	async method() {}
```

Comment:
It guess it's a regression.
[Prettier playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAucAbAhgZ0wAgGIQQCMOwAOlGTFgJ5Rg4D0AVOgGbwBOzjOAtnBgALCABMAFAEpSAXwoyQAGhAQADjACW0TMlDpOnCAHcACvoQ6U6VEfQ0dygEad0YANaCAyugEAZDVBwyGzWmHBOLu5eqq4BAObIMJwAruEgYXwaiSlpcAAeqnCcGgKw1gAqRVD6GnCWIahhypjxqHAAiskQ8MGhaQBWmHmerR1dPUgNTSAAjuNwJoaqliBYALSBcKJbSiBJ6Bqo8QDCEHx86MirqKi7LVBxbQCCMEkajsnwJkX+gb2NaSEMD4qAA6kINPBMDEwHBPBZIRoAG6QmhXMDYXZI1IASSg21gnjAxXUT3xnhgNDa-2mqkMYVBLlUVzpdSKSKCygCYU4MEW6DiFxpaRinB5V0phUwxI06l2dICMFBGlEwmQAA4AAzKThwOYaXX8wWXSZ9ZTURzK1VCZAAJmUyTC5XQjnqZpAcD4ji221EvnQD2SArgBE4F1e8Su6E+EBAMhkQA)

Reply:
Sorry,  you're right(: